addons/mrp/views/mrp_workorder_views.xml

```diff
@@ -404,28 +404,28 @@
                 <field name="working_state"/>
                 <field name="workcenter_id"/>
                 <field name="product_id"/>
-                <field name="qty_production"/>
+                <field name="qty_remaining"/>
                 <field name="product_uom_id" force_save="1"/>
                 <templates>
                     <t t-name="kanban-box">
                         <div class="oe_kanban_global_click">
                             <div class="o_kanban_card_header o_kanban_record_top">
                                 <div class="o_kanban_record_headings o_kanban_card_header_title">
                                     <strong class="o_primary">
-                                        <span><t t-esc="record.production_id.value"/></span> - <span><t t-esc="record.name.value"/></span>
+                                        <span><t t-out="record.production_id.value"/></span> - <span><t t-out="record.name.value"/></span>
```

Comment:
You can take the opportunity to shorter a little the template, as you can write the `t-out` directly in the html tag:
```suggestion
                                        <span t-out="record.production_id.value"/> - <span t-out="record.name.value"/>
```

Reply:
Changed as suggested in 060cc2d